app/styles/ui/window/_app-menu-bar.scss

```diff
@@ -42,5 +42,15 @@
       background: var(--background-color);
       pointer-events: all;
     }
+
+    .menu-item:hover {
+      &:not(.disabled) {
```

Comment:
You should be able to merge these selectors together.

Reply:
👍 It took me a fair bit to figure out the right syntax here, so I think leaving it like this for future peeps is a safe bet.